python/pyspark/__init__.pyi

```diff
@@ -72,4 +72,4 @@ def copy_func(
 ) -> F: ...
 def keyword_only(func: F) -> F: ...
 
-__version__: str
+from pyspark.version import __version__ as __version__
```

Comment:
Nit-pick: Could you move this to the top of the file, next to the other imports?

Reply:
oops, done. : )